shenyu-admin/src/main/java/org/apache/shenyu/admin/model/bean/UpstreamInstance.java

```diff
@@ -156,4 +158,26 @@ public boolean isEnabled() {
     public void setEnabled(final boolean enabled) {
         this.enabled = enabled;
     }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
use `Objects.isNull(o)` replace `o == null`

Reply:
ok i will replace it